azureDatabaseUtils/src/main/java/bio/terra/workspace/azureDatabaseUtils/runners/PgDumpDatabaseRunner.java

```diff
@@ -0,0 +1,48 @@
+package bio.terra.workspace.azureDatabaseUtils.runners;
+
+import bio.terra.workspace.azureDatabaseUtils.database.DatabaseService;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.boot.ApplicationArguments;
+import org.springframework.boot.ApplicationRunner;
+import org.springframework.context.annotation.Profile;
+import org.springframework.stereotype.Component;
+
+@Profile("PgDumpDatabase")
+@Component
+public class PgDumpDatabaseRunner implements ApplicationRunner {
+
+    @Value("${env.workflowCloning.sourceDbName}")
```

Comment:
I put all of the environment variables needed under a single namespace, but I suspect I can reuse some of the existing ones